source/MRViewer/MRRibbonButtonDrawer.cpp

```diff
@@ -78,11 +78,17 @@ bool RibbonButtonDrawer::GradientCheckbox( const char* label, bool* value )
 {
     auto& texture = GetGradientTexture();
     if ( !texture || ( value && !*value ) )
-        return ImGui::Checkbox( label, value );
+    {
+        ImGui::PushStyleVar( ImGuiStyleVar_FramePadding, { 3.0f, 3.0f } );
```

Comment:
you should use menu_scaling, not raw values and i think we should not change sizes in common code, better do it in plave where we realy need it (different calls may need different padding)